### DISCUSSION TITLE
Scrollable pane

### DISCUSSION THREAD

**Discussion by robertjmccarthy** [ministryofjustice/moj-frontend#711] [RELEASED: Components, pages and patterns]
The [scrollable pane component](https://design-patterns.service.justice.gov.uk/components/scrollable-pane/) is live in the MoJ Design System.

Use this discussion to suggest changes and improvements to this component.

**Comment by manalishi79** [comment]
We've been using a scrollable pane on an internal product for financial data at DBT. The product reproduces the format of spreadsheets which the users are comfortable with from working in finance.

In testing we found that less experienced users often miss content on the far right (off screen). We've got UI at the top to hide columns which can narrow the view. This seems a reasonable suggestion to improve things - our challenge now is users missing the show/hide option above the table.

<img width="1202" alt="Screenshot 2025-06-06 at 16 41 21" src="https://github.com/user-attachments/assets/8ec800b5-2206-43a1-b2ba-9b7f9f671e34" />

**Comment by msg4matt** [comment]
@robertjmccarthy We are looking at using the scrollable pane for a new service and I noticed that there is some invalid css to show a right and left gradient - to indicate there is more information to be seen.

![image](https://github.com/user-attachments/assets/fa0f5fb4-2770-4037-b6bc-912ae1e9aaa2)

To get this to work correctly, I modified the css to:

`linear-gradient(to right,#fff,#fff,#fff0 calc(.75em*2)),
radial-gradient(farthest-side at 0 50%,#0003,#fff0),
linear-gradient(to left,#fff,#fff,#fff0 calc(.75em*2)),
radial-gradient(farthest-side at 100% 50%,#0003,#fff0) 100%`

**Comment by chrispymm** [comment]
The scrollable pane applies a background to the container meaning this only works on white backgrounds. 

Need to investigate if it can be made to work with transparent backgrounds to be more flexible.
